ouroboros-network/test/Test/Ouroboros/Network/Testnet/Simulation/ConnectionManager.hs

```diff
@@ -166,6 +166,36 @@ verifyAbstractTransition Transition { fromState, toState } =
 
       _ -> False
 
+-- Assuming all transitions in the transition list are valid, we only need to
+-- look at the 'toState' of the current transition and the 'fromState' of the
+-- next transition.
+verifyAbstractTransitionOrder :: [AbstractTransition]
```

Comment:
This function should also be imported from somewhere in `ouroboros-network-framework` (the testing library).

Reply:
Done